src/components/buttons/Button.vue

```diff
@@ -180,6 +182,10 @@ const usedElevation: ComputedRef<number | string> = computed(() => {
     &.sm {
       @apply px-2 py-2;
     }
+
+    &.text {
+      @apply disabled:bg-transparent disabled:active:bg-transparent #{!important};
```

Comment:
`fab` and `text` is unified in variant now. So they can have both class. Then this line will not be used at all.

Reply:
okay, on it, will also rename `type` to variant in progress and stepper